src/client/components/language-selector/language-selector.vue

```diff
@@ -1,13 +1,15 @@
 <template>
```

Comment:
Looks a lot better than before. Okay for now. We need to spec out the language-selector and figure out how it can work with 5 or 10 languages to pick from.

Reply:
The language selector was only styled enough so the header looks good. Since it has a separate ticket on Trello, I'm working on it on a separate branch.